src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Class representing a User
+public class User {
+  private long id;
```

Comment:
user ID will have to be changed to a String, google sign-in API's getID method returns as an opaque String.